### ISSUE TITLE
[Sozialpass] Add multiple cards

### ISSUE THREAD

**Issue by f1sh1918** [digitalfabrik/entitlementcard#1084]
As a user I want to be able to add multiple id cards to my device

## Acceptance Criteria:
 - until 5 id cards can be added
 - each card can be deleted separately
 - it should be possible to navigate between cards and verify each of them

## Technical Tasks:
 - add a button to delete a card
 - add a view where you swipe our navigate between multiple cards
 - adding a new card should not override an old card (as it is now)
 - each card should be verified when the app starts (self verification)
 - if 5 cards are already activated show an alert if the user wants to add another card instead of opening the camera
 
 Small draft for the swipe area was added (has not to be used for design)

![swipe-area](https://github.com/digitalfabrik/entitlementcard/assets/37902063/f0f063d3-5f3c-4acf-ad0b-d29105546752)

**Comment by f1sh1918** [comment]
blocked by #1085
